test/validate/windows_validate.go

```diff
@@ -78,6 +79,29 @@ var windowsChecksMap = map[string][]check{
 			cmd:              cnsWinCachedAssignedIPStateCmd,
 		},
 	},
+	"StatelessCni": {
```

Comment:
We need to add validation check for CNS statefile.

Reply:
Will copy th linux cns validation here